### PR TITLE
Allow context reuse after close

### PR DESCRIPTION
**Pull request by medusalix**
Works on Windows 😃. Not sure about the other platforms.

Related to #47

### CODE REVIEW

**Comment by hajimehoshi** [comment]
Sorry to keep you waiting. I'm now pretty busy and I'll take a look this PR in this week. Thanks.

**Comment by medusalix** [comment]
No worries 😀. The old version is still working fine.

File: context.go [hunk 1]
```diff
@@ -103,6 +103,8 @@ func (c *Context) NewPlayer() *Player {
 // Close closes the Context and its Players and frees any resources associated with it. The Context is no longer
 // usable after calling Close.
 func (c *Context) Close() error {
+	theContext = nil
```
Comment (hajimehoshi) [inline context.go]:
```go
contextM.Lock()
theContext = nil
contextM.Unlock()
```

**Comment by hajimehoshi** [comment]
I have not tested this on other machines, but removing this should not be problematic. lgtm, thanks!

**Pull request merged**